### DISCUSSION TITLE
Display Time option needs zero as a possible value

### DISCUSSION THREAD

**Discussion by KodeXall** [jonaskohl/CapsLockIndicator#133] [Ideas]
Right now a minimum number of milliseconds that can be set at

**_Notification > Display time_**

is 55. I do not understand why user should not be allowed to choose just 0 seconds

If I am not mistaken, a new possibility of setting of zero in conjunction with all other currently available options would be the only way for a user to not see a Notification of an ON or OFF state, with the opposite state being shown indefinitely. In other words: it is impossible now to e.g. see constantly the "Caps Lock turned on" Notification but without having to experience even a brief flash of the "Caps Lock turned off" Notification when turning it off

I myself all I need is just a clear Notification about CapsLock being ON - while info about CapsLock being OFF is just distracting to me. And a workaround in form of going to

**_Advanced Options > Advanced Settings > customMessageCapsOff_**

and changing it to match the **_String_** of **_customMessageCapsOn_** would work - but could backfire when forget about and when trying to diagnose some issue and / or changing the colors of Notifications

**Comment by jonaskohl** [comment]
This sounds reasonable. I will consider this for a future update
